docker-app/qfieldcloud/core/tests/test_api.py

```diff
@@ -36,9 +37,30 @@ def setUp(self):
 
     def test_api_status(self):
         response = self.client.get("/api/v1/status/")
+
         self.assertTrue(status.is_success(response.status_code))
-        self.assertEqual(response.json()["storage"], "ok")
-        self.assertEqual(response.json()["geodb"], "ok")
+
+        data = response.json()
+
+        self.assertIn("database", data)
+        self.assertEqual(data["database"], "ok")
+
+        self.assertIn("storage", data)
+        self.assertEqual(data["storage"], "ok")
+
+        self.assertIn("storages", data)
+        self.assertIsInstance(data["storages"], dict)
+
+        expected_storages = storages.backends
+        # here we don't care about the configured ManifestStaticFilesStorage.
+        expected_storages.pop("staticfiles")
+
+        found_storages = data["storages"]
+
+        self.assertEqual(expected_storages.keys(), found_storages.keys())
```

Comment:
Shall we sort them or convert to set, otherwise if we have a different order of the keys the test will fail?